components/script/dom/servoparser/mod.rs

```diff
@@ -585,6 +585,9 @@ impl FetchResponseListener for ParserContext {
                 }
             },
             Some(ContentType(Mime(TopLevel::Text, SubLevel::Xml, _))) => {}, // Handle text/xml
+
+            Some(ContentType(Mime(TopLevel::Application, SubLevel::Xml, _))) => {}, // Handle application/xml
```

Comment:
I don't think adding a new match arm here is the correct move; the issue says that `application/xml` should be treated the same as `text/xml`, so both enum variants should be on the same match arm.